flink-connectors/flink-connector-kinesis/src/main/resources/META-INF/NOTICE

```diff
@@ -12,15 +12,11 @@ This project bundles the following dependencies under the Apache Software Licens
 - com.amazonaws:jmespath-java:1.11.319
 - org.apache.httpcomponents:httpclient:4.5.3
 - org.apache.httpcomponents:httpcore:4.4.6
+- com.amazonaws:amazon-kinesis-client:1.10.0
```

Comment:
my "mvn dependency:tree" shows

```
[INFO] +- com.amazonaws:amazon-kinesis-client:jar:1.11.2:compile
[INFO] |  +- com.amazonaws:aws-java-sdk-dynamodb:jar:1.11.603:compile
[INFO] |  |  \- com.amazonaws:aws-java-sdk-s3:jar:1.11.603:compile
[INFO] |  |     \- com.amazonaws:aws-java-sdk-kms:jar:1.11.603:compile
[INFO] |  \- commons-logging:commons-logging:jar:1.1.3:compile
```

thus, are we missing `aws-java-sdk-dynamodb`, `aws-java-sdk-s3`, and `aws-java-sdk-kms`?

Reply:
most likely, there were a few sub-dependencies missing in the notices and i was unsure of where they should be listed. I'll add these in here.